### PR TITLE
Correct output parameter in cloudaccess example

### PR DESCRIPTION
**Pull request by mrsheepuk**

### CODE REVIEW

File: modules/cloudaccess/examples/complete/outputs.tf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 output "managed_identity_client_id" {
   value = module.wayfinder_cloudaccess.managed_identity_client_id
 }
-output "managed_identity_principal_id" {
-  value = module.wayfinder_cloudaccess.managed_identity_principal_id
+output "managed_identity_tenant_id" {
```
Comment (KashifSaadat) [inline modules/cloudaccess/examples/complete/outputs.tf:4]: Need to autogenerate / update the readme file for this.

**Review by KashifSaadat — CHANGES_REQUESTED** [review]

**Comment by KashifSaadat** [comment]
@gambol99 looks like there has been a breaking CI workflow change?

```
The workflow is not valid. .github/workflows/terraform-module-validation.yml (Line: 17, Col: 11): Input enable-terraform-tests is required, but not provided while calling.
```

File: modules/cloudaccess/examples/complete/outputs.tf (hunk 1 above)
Comment (mrsheepuk) [reply modules/cloudaccess/examples/complete/outputs.tf:4]: There is no README for the example.......... 👀 😞 what do I need to run?

Comment (KashifSaadat) [reply modules/cloudaccess/examples/complete/outputs.tf:4]: Try this: `terraform-docs markdown table --output-file ${PWD}/README.md --output-mode inject .`

Comment (KashifSaadat) [reply modules/cloudaccess/examples/complete/outputs.tf:4]: The CI workflow isn't validating this examples dir so ignore

**Review by KashifSaadat — APPROVED** [review]

**Pull request merged**